README.md

```diff
@@ -0,0 +1,67 @@
+Designed to be used as a Vue component: `Vue.component(element, Calus)` where element is the name of the template i.e. 'calendar-calus'. You can then use:
+```html
+<calendar-calus inline-template></calendar-calus>
+```
+
+### Props
+You can pass the following props from the template:
+- `time-zone`: defaults to local otherwise can be explicitly set (i.e. Europe/London)
+- `available-dates`: list of dates which are available to select
+- `display-in-column`: whether to show all the months in a column, or a single month with controls to change which month is shown
+- `linear-dates`: linear view for when showing a free flowing calendar
+- `week-starts-on-sunday`: force the day to start on sunday instead of default monday
+- `on-select`: callback for when an available date is clicked
+- `on-change-month`: callback for when selected month is changed with button
+- `allow-previous-scroll`: defaults to false, set to true if you want to be able to scroll to months previous to today's
+
+### Methods
+- `select()`: method to call when clicking on a date
+- `setAvailable()`: pass array of date strings where an event is available
```

Comment:
As discussed mark these methods as Deprecated as they are handled via `props`
Also if you want to have `select()` method it should select a date in the calendar as opposed to accepting a callback function for when a date is selected

Reply:
`select()` isn't actually deprecated - it calls the function onSelect which is passed in via props but also handles styling changes i.e. changing class names. `select()` is used as an on-click on a day in the template `select(day)`